pyogrio/raw.py

```diff
@@ -2,11 +2,18 @@
 import os
 
 from pyogrio._env import GDALEnv
+from pyogrio.errors import DataSourceError
 from pyogrio.util import get_vsi_path
 
 with GDALEnv():
     from pyogrio._io import ogr_read, ogr_read_arrow, ogr_write
-    from pyogrio._ogr import remove_virtual_file, _get_driver_metadata_item
+    from pyogrio._ogr import (
+        get_gdal_version,
+        get_gdal_version_string,
```

Comment:
```suggestion
        get_gdal_version,
```

(the latter is not used here?)

Reply:
Both are used here; using the string version on line 289 for the error message